source/PlanetLabel.cpp

```diff
@@ -86,7 +88,7 @@ PlanetLabel::PlanetLabel(const Point &position, const StellarObject &object, con
 	const Planet &planet = *object.GetPlanet();
 	name = planet.Name();
 	if(planet.IsWormhole())
-		color = Color(.8f, .3f, 1.f, 1.f);
+		color = *planet.GetWormhole()->GetLinkColor();
```

Comment:
It should be noted that you're changing the default planet label colour for wormholes.